activerecord/test/cases/associations/belongs_to_associations_test.rb

```diff
@@ -23,6 +23,9 @@
 require 'models/ship'
 require 'models/treasure'
 require 'models/parrot'
+require 'models/moon'
+require 'models/planet'
+require 'models/star'
```

Comment:
Yes I will change that.  I thought it would be easier to maintain all the tests if they didn't share models, but I can see how large number of models doesn't look so good.